content/chat/rooms/messages.textile

```diff
@@ -0,0 +1,80 @@
+---
+title: Messages
+meta_description: "Send and receive messages in chat rooms."
+product: chat
+languages:
+  - javascript
+---
+
+You can send and receive messages in a chat room with any number of participants. These users subscribe to messages by registering a listener, and send messages to all users that are subscribed to receive them.
+
+h2(#subscribe). Subscribe to messages
+
+Subscribe to receive messages in a room by registering a listener. Use the "@messages.subscribe()@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/Messages.html#subscribe method in a room to receive all messages that are sent to it:
+
+```[javascript]
+const {unsubscribe} = room.messages.subscribe(message) => {
+  console.log(message);
+}
+```
+
+h3(#payload). Message payload
+
+The following is an example payload of a message:
+
+```[json]
+{
+  "timeserial": "e91RB0GOQBcvyJ53053074@1718191654749-0",
+  "clientId": "basketLover014",
+  "roomId": "basketball-stream",
+  "text": "What a shot!",
+  "headers": {},
+  "metadata": {},
+  "createdAt": "2024-06-12T11:37:59.988Z",
```

Comment:
This is a javascript `Date` object and not a string. Can't be represented in JSON. Not sure how to make it not confusing.

Reply:
Mmm, yes true. I think this should be evident with the type 'Date' in the table below though? WDYT?

Reply:
That's good!